src/shared/utils/__tests__/cleanText.spec.ts

```diff
@@ -7,7 +10,7 @@ const pairs: Array<[input: string | null | undefined, output: string]> = [
   ['<a>link</a>', 'link'],
   /* Can't access the css modules values from tests, so no classes... 😕 */
   // ['<h1>title</h1>', '<strong class="heading">title</strong>'],
```

Comment:
Should probably update the comment accordingly too, even if it's not a testable use-case